### ISSUE TITLE
1.0.3 missing built files

### ISSUE THREAD

**Issue by Jarred-Sumner** [alexZajac/react-native-skeleton-content-nonexpo#1]
It appears that 1.0.3  published in NPM is missing the `lib` folder, making this version of the package unusable. Is there any chance you could republish it?

```
> cd node_modules/react-native-skeleton-content-nonexpo/              (base)
> ls                                                                  (base)
  LICENSE.md    package.json    README.md
```

**Comment by itsam** [comment]
I second this. Could you plz republish? Thanks in advance

**Comment by alexZajac** [comment]
Hi @itsam and @Jarred-Sumner ! 
I solved that issue in the 1.0.4 release, thank you.
